terraform/tags.tf

```diff
@@ -0,0 +1,11 @@
+module "beacons_label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
```

Comment:
Looking at the docs as well, this should be tied to a specific release rather than `master`.

See: 
![image](https://user-images.githubusercontent.com/76042279/115395170-7bb06680-a1db-11eb-9844-5e94dc8dea44.png)

And docs: https://registry.terraform.io/modules/cloudposse/label/null/latest#usage

Reply:
Great catch 👌🏻  Done.